src/platform/nrf_pca10028/src/systick.rs

```diff
@@ -0,0 +1,134 @@
+//! Stub implementation of a systick timer since the NRF doesn't have the
```

Comment:
This belongs in the `nrf51` crate instead. That way, the implementation for the `main::Chip` trait's `SysTick` associated type. At the moment this isn't actually doing anything :(